Kernel/init.cpp

```diff
@@ -458,4 +461,31 @@ void setup_pci()
     PCI::Initializer::the().dismiss();
 }
 
+void setup_interrupts()
+{
+    if (!KParams::the().has("smp")) {
+        InterruptManagement::initialize();
+        return;
+    }
+    auto smp = KParams::the().get("smp");
+    auto* madt = (ACPI_RAW::MADT*)ACPIParser::the().find_table("APIC");
+    if (smp == "off") {
+        InterruptManagement::initialize();
+        return;
+    } else if (smp == "on") {
```

Comment:
No need for `else` after `return`. Also, the same comment repeated everywhere else relevant. :)